src/app/pages/quickMaps/pages/baselineDetails/chartCard/chartCard.component.ts

```diff
@@ -54,6 +57,56 @@ export class ChartCardComponent implements OnInit {
     },
   };
 
+  // histo
+  public histograph = {
```

Comment:
This needs to be an Interface, and if gets more complex with methods, a class (https://stackoverflow.com/questions/54356711/when-to-use-class-or-interface-in-angular-project)